node-graph/graph-craft/src/document/value.rs

```diff
@@ -62,6 +62,7 @@ pub enum TaggedValue {
 	Artboard(graphene_core::Artboard),
 	IVec2(glam::IVec2),
 	SurfaceFrame(graphene_core::SurfaceFrame),
+	Palette(Vec<Color>),
```

Comment:
Is that what have you mean about the type Palette?

Reply:
This enum is used for serialising constant inputs in the graph (as `Box<dyn DynAny>` cannot be serialised). This will be useful in future when we add nodes that use the colour palette.